### DISCUSSION TITLE
Custom domain issue and duplicate deployments

### DISCUSSION THREAD

**Discussion by tsplude** [vercel/community#388] [Help] [answered]
Hi all,

I'd like to transfer the hosting of my domain to Vercel. Steps I've taken:
  - Upgraded my account from "hobby" to "pro"
  - Set up project within this pro "team" which mirrors my original project in my personal account
  - Followed [this](https://vercel.com/docs/concepts/projects/custom-domains) guide for setting up my custom domain
    - All I did was adjust the `A` record on GoDaddy to be `A/@/76.76.21.21` and then both `mysite.com` and `www.mysite.com` flipped to `Valid` as per [step 5](https://vercel.com/docs/concepts/projects/custom-domains#step-5:-domain-configured)

Questions:
  - Nothing is different on my domain, the changes in my deployment are not reflected when I visit the page. Have I missed something? Does this process usually take hours?
  - Every time I push to `main` now I get duplicate deployments because the project exists on both my team and my personal account - how do I disable the project on my personal account? Am I supposed to delete my personal account now that I have a "pro" team account?

Thanks

**Comment by tsplude** [comment, marked as answer]
First question is fixed :)

Still wondering how to disable a project in my personal account

**Reply by leerob** [reply]
Yep, you can now delete the other project, which will stop the duplicate deployments!

**Reply by tsplude** [reply]
For posterity, to delete a specific project and not the personal account as a whole I had to go to `Project Settings -> Advanced -> Delete Project`
